python/lib/dependabot/python/update_checker/pip_compile_version_resolver.rb

```diff
@@ -91,12 +91,18 @@ def fetch_latest_resolvable_version_string(requirement:)
         def compile_file(filename)
           # Shell out to pip-compile.
           # This is slow, as pip-compile needs to do installs.
+          pip_compile_command = "pyenv exec pip-compile"
+          if (requirements_file = compiled_file_for_filename(filename)) &&
+             requirements_file.content.include?("autogenerated by uv")
+            pip_compile_command = "pyenv exec uv pip compile"
+          end
```

Comment:
These lines seem to be duplicated by 210-214 so it might be worth refactoring into a helper function. That might also make the linter happy.

Reply:
Did that, only reduced one line lol. Other ideas? Not my main language this ruby stuff, sorry, trying my best from my phone because I cannot sleep with this pr open.